### PR TITLE
Update website info

### LINKED ISSUE #14: Update website info
Update the information in the public folder and README.md

### PR DESCRIPTION
**Pull request by ss8651twtw**
Fix #14

### CODE REVIEW

**Comment by ss8651twtw** [comment]
Need to update the logo.

**Comment by ss8651twtw** [comment]
image

Cool!

**Review by fwpp — COMMENTED** [review]
Great! Image fits ramen, cool! There is only a place to adjust.

File: public/logo192.png
[the remark this answers is not in the document]
Comment (fwpp) [reply public/logo192.png:1]: May you put commit 25821cde471b0fef2cdbb46186a7f6c335a0f803 before 7beaf7e41e2f179276bbd97644113c6b57ded87c? In commit 7beaf7e41e2f179276bbd97644113c6b57ded87c, 'public/apple-touch-icon.png' doesn't exist.

[the remark this answers is not in the document]
Comment (fwpp) [reply public/logo192.png:1]: Thanks!

**Review by fwpp — APPROVED** [review]

**Pull request merged**